static/app/data/forms/replay.tsx

```diff
@@ -24,6 +24,9 @@ const formGroups: JsonFormObject[] = [
         label: t('Create Hydration Error Issues'),
         help: t('Toggles whether or not to create Session Replay Hydration Error Issues'),
         getData: data => ({options: data}),
+        visible({features}) {
+          return features.has('session-replay-hydration-error-issue-creation');
+        },
```

Comment:
I was looking at https://github.com/getsentry/sentry/blob/6223dd4edd7f5a479ba25c9b7b16ffdae77e2072/static/app/data/forms/projectSecurityAndPrivacyGroups.tsx#L38 for an example. Not 100% sure though it's getting passed along... it'll be easier to check once https://github.com/getsentry/sentry-options-automator/pull/1540/files is deployed